tests/e2e/__init__.py

```diff
@@ -23,9 +22,6 @@ def setUp(self):
         self.GOOD_REFERER = 'http://unittest.geo.admin.ch'
```

Comment:
Do we still use referrer somewhere? If not, may an additionnal cleanup PR

Reply:
yes, i think the GOOD_REFERER, BAD_REFERER can be removed too.